src/pandas_profiling/model/pandas/correlations_pandas.py

```diff
@@ -73,12 +73,15 @@ def pandas_cramers_compute(
 ) -> Optional[pd.DataFrame]:
     threshold = config.categorical_maximum_correlation_distinct
 
-    categoricals = {
-        key
-        for key, value in summary.items()
-        if value["type"] in {"Categorical", "Boolean"}
-        and value["n_distinct"] <= threshold
-    }
+    # Index must not be a set, so sort it into a list
```

Comment:
Can you please explain why the sorting is required with pandas 1.5 support?

Reply:
Sure:

* https://github.com/pandas-dev/pandas/commit/cde13db86a2
* https://github.com/pandas-dev/pandas/pull/47231
* https://github.com/pandas-dev/pandas/issues/47215

Reply:
So the sorting is not related to the support for pandas 1.5. Just a nice to have based on my understanding.

Reply:
It is related to 1.5 - the PR linked above that prevents using sets for indexes only landed in 1.5.

It is possible the order changes, but then again that used to be the case prior to this change, since sets are inherently unordered.

If you like, I can change `sorted` to `list` instead, to preserve the possibly arbitrary order the set is iterated in.

Reply:
Just wanted to double-check whether this was something new, or a match of former functionality. 

Thank you for the clarification! Yes, that was my understanding and I would prefer to preserve for now the inherited arbitrary order.

Reply:
Updated to use a plain `list()`, with an explanatory comment. 👍